rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py

```diff
@@ -70,6 +77,14 @@ def getImageUploadAndColor():
     im = Image.open(file_path)
     im.show()
 
+    endPts = {
+        "color":colorRequest,
+        "image":imageRequest,
+        "userHeight":userHeightRequest,
+        "wallHeight":wallHeightRequest
+    }
```

Comment:
Same here as I commented in the other endpoint pr. If not appearing then do not send back and space after :, along with the comment put here
https://github.com/ARC-Summer-2020/AutoRockClimber/pull/16/files#r480688710